src/goInstallTools.ts

```diff
@@ -380,27 +380,42 @@ function getMissingTools(goVersion: SemVersion): Promise<string[]> {
 }
 
 // If langserver needs to be used, but is not installed, this will prompt user to install and Reload
-// If langserver needs to be used, and is installed, this will return true
-// Returns false in all other cases
-export function checkLanguageServer(): boolean {
+// If langserver needs to be used, and is installed, this will return the list of supported flags
+// Returns null in all other cases
+export function checkLanguageServer(): string[] {
```

Comment:
To keep things simple, lets go with the assumption that the auto-completion feature from language server is available only if the user has the latest language server i.e if the user's version of the language server respects the `initalizationOptions` sent.

Towards that end, we can revert the changes made to `checkLanguageServer` here